### ISSUE TITLE
Re-Evaluate HTTPS stuff

### ISSUE THREAD

**Issue by MaStr** [PirateBox-Dev/PirateBoxScripts_Webserver#131]
We have it currently not enabled, but https-everywhere forces us to react.

Possible actions are:
  * Ship an active preconfiguration for lighttpd
  * Ship an inactive optional configuration for lighttpd
  * Install/make use of a captive portal

**Comment by MaStr** [comment]
http://www.andybev.com/index.php/Using_iptables_and_PHP_to_create_a_captive_portal
https://wiki.openwrt.org/doc/howto/wireless.hotspot.nodogsplash

**Comment by TheExpertNoob** [comment]
I see we're working on captive portal files, or is that out of date?... Also, maybe.... make a schema... or folder/*.schema that runs through all instances of ########HOST######### for piratebox.lan. As I see it referenced in the ia_handler.php and www_content/index.html for droopy(the script may need to modify  /opt/piratebox/share/content/index.html).

**Comment by MaStr** [comment]
right, I rewrote our fake internet detection answers.  This is mostly covered by #58 
btw: ia_handler.php points to piratebox.lan which then uses index.html to redirect to the correct custom domain (yes, might still cause issues- mh)
